codejail_service/settings/base.py

```diff
@@ -45,8 +41,13 @@ def root(*path_fragments):
 # Python dotted path to the WSGI application used by Django's runserver.
 WSGI_APPLICATION = 'codejail_service.wsgi.application'
 
-# No database for codejail
-DATABASES = {}
+# Django *really* wants a database. So we give it a blank in-memory one.
```

Comment:
Do we want to add any sort of warning in the README that the service should be treated as-if it has no database?

Reply:
It's mentioned in the README -- do you think it should be more prominent?

Reply:
Yes. And I don't just mean that the service does not have a database, but a warning about the required db for Django. For example, something like:
```
Warning: This service is configured with an in-memory database simply to make Django happy. The service itself should be considered to not to have a database, and should not rely on any database-dependent features, like waffle-based toggles, as one example.

Reply:
Ah, I see. Made some changes, take a look.